app/commands/payments/subscription/create.rb

```diff
@@ -22,6 +22,7 @@ def call
         User::UpdateActiveDonationSubscription.(user)
       when :premium
         User::Premium::Update.(user)
+        User::InsidersStatus::Update.(user)
```

Comment:
I don't understand this line. Creating a Premium subscription shouldn't affect someone's insider status should it?

Reply:
No that line can be removed. It was a test for the flair bit